packages/admin-ui-docs/docs/components/checkbox.mdx

```diff
@@ -511,31 +532,157 @@ function Example() {
+  </ul>
+</p>
+</details>
+
+<details><summary>When to use a Checkbox Group instead of a Checkbox?</summary>
+<p>Since the Checkbox label should only correspond to its value, the choice should be contextualized using the label of a Checkbox Group or the title of a form section. This is why the Checkbox Group is usually the right choice even for binary choices where there is only a single Checkbox.</p>
+</details>
+
+<details><summary>When to use a vertical or a horizontal Checkbox Group?</summary>
+<p>The vertical option is more scalable, but the horizontal option can be used when, considering localization, options don't require a line break and the length of their label does not vary a lot between options, including up to 20 characters.</p>
+</details>
+
+</PageSection>
+
+<PageSection
+  id={tabToc.designReference[1].id}
+  title={tabToc.designReference[1].value}
+>
+
+<details><summary>How should a Checkbox component be positioned in a form?</summary>
+<p>Prefer placing a Checkbox at the start or end of a form section.</p>
+</details>
+
+</PageSection>
+
+<PageSection
+  id={tabToc.designReference[2].id}
+  title={tabToc.designReference[2].value}
+>
+
+<details><summary>When should a Checkbox be disabled?</summary>
+<p>
+  <ul>
+    <li>Use a disabled Checkbox only if in some condition it becomes enabled. Example: when an option is incompatible with another option in a form.</li>
+    <li>Use a <a href="https://admin-ui.vercel.app/components/tooltip">tooltip</a> trigger to explain why the Checkbox is disabled. Display the information when hovering over the tooltip trigger — not the Checkbox itself.</li>
+  </ul>
+</p>
+</details>
+
+<details><summary>When should the indeterminate state be used?</summary>
+<p>
+In scenarios where it might be necessary to check or uncheck all options, a Checkbox can be placed in the beginning of the group with the <em>Select all</em> label. Make sure options below are nested and vertically aligned with the <em>Select all</em> label.<br /><br />
+The indeterminate state should be displayed in these scenarios when only a few options (not all) are checked. When clicking the indeterminate state, all options must be checked. When clicking for a second time, all options must be unchecked.
+</p>
+</details>
+
+<details><summary>What should be the default state of a Checkbox?</summary>
+<p>
+When a Checkbox serves as confirmation, leave it unchecked by default and make it required to direct the attention of merchants. For example, the choice to accept terms and conditions.
+</p>
+</details>
+
+</PageSection>
+
+<PageSection
+  id={tabToc.designReference[3].id}
+  title={tabToc.designReference[3].value}
+>
+
+<details><summary>What should be the label of a Checkbox?</summary>
+<p>
+  <ul>
+    <li>Write the label so it corresponds to the value of the Checkbox.</li>
+    <li>Use sentence case but capitalize proper nouns.</li>
+  </ul>
+  <br /><br />
+  What to avoid:
+  <ul>
```

Comment:
Adicionar esse ponto:
Don't write labels that vary a lot in length between options.